client/src/components/dashboard/Dashboard2.js

```diff
@@ -1,59 +1,59 @@
-import React, { Component } from 'react';
```

Comment:
we need to have the same formatting setting or else this is going to keep happening!